### PR TITLE
docs: Add a Code of Conduct

### PR DESCRIPTION
**Pull request by eranhirsch**
Using the template provided by GitHub.

Set my personal email as the point of contact for abuse reports.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+eranhirsch@gmail.com.
```
Comment (eranhirsch) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
eranhirsch+remeda@gmail.com.
```

**Review by eranhirsch — COMMENTED** [review]
better

**Review by lstkz — APPROVED** [review]

**Review by cjquines — COMMENTED** [review]
sgtm. as a principle i think there should always be at least two contact methods in case someone is uncomfortable reporting to one person, but this is fine for now

**Pull request merged**